core-bundle/assets/backend.js

```diff
@@ -49,8 +49,8 @@ const mooDomready = () => {
     if (!document.body.mooDomreadyFired) {
         document.body.mooDomreadyFired = true;
 
-        if (Element.Events.removeEvents) {
-            Element.Events.removeEvents();
+        if (window.removeEvents) {
+            window.removeEvents();
```

Comment:
Also works with `document.body.removeEvents()` but since we do `window.fireEvent()` on line 56, I figured `window` would be correct?

Reply:
I have no idea

Reply:
If it fixes the problem, let's still go with this, I'd say. We need to switch to another datepicker in the near future anyways.